cvmfs/CMakeLists.txt

```diff
@@ -354,10 +354,7 @@ if (BUILD_SERVER)
     add_dependencies (geoip python-geoip)
   endif (GEOIP_BUILTIN)
 
-  include (${VJSON_BUILTIN_LOCATION}/CVMFS-CMakeLists.txt)
```

Comment:
It is included in the top-level CMakelist now (https://github.com/cvmfs/cvmfs/pull/975/files#diff-af3b638bc2a3e6c650974192a53c7291R336), so there are no clashes